Developers-Notes/CONP_data_upload_admin.md

```diff
@@ -0,0 +1,45 @@
+
+Log in to ```sftp.conp.ca``` as user ```proftpd```.  If you need the password for this account, contact @cmadjar or @emmetaobrien.
```

Comment:
```suggestion
Log in to ```sftp.conp.ca``` as user ```proftpd```.  If you need the password for this account, contact Cécile Madjar or Emmet O'Brien.
```

The @cmadjar and @emmetaobrien does not redirect to a link. We could either put our mcin emails or just our names.

Reply:
Before this sentence, I would add a title: Admin login information to the SFTP server (or something like that)